test/unit/seed.js

```diff
@@ -391,7 +394,7 @@ describe("Contract: Seed", async () => {
         //                 alternativeSetup.token.fundingToken.address
         //             ],
         //             [softCap, hardCap],
-        //             price,
+        //             priceAndSeedTokenDecimal,
```

Comment:
Just curious why this whole section is commented out? Altho it isn't done in this PR, maybe we can uncomment or remove it :) Also do you maybe know if we have an equivalent test somewhere else?